server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -198,10 +204,10 @@ public void canGetAllUsers() throws IOException {
   public void canGetUsersWithAge37() throws IOException {
```

Comment:
I think my intent had been to get rid of this _in favor of_ the (then new) `getUsersByAge()` method. Is there a reason to keep this? If we do, students will just copy and paste it all over Lab 4 and then their projects.